cmd/av/pr_queue.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/avgql"
+	"github.com/shurcooL/graphql"
+	"github.com/spf13/cobra"
+)
+
+var prQueueFlags struct {
+	SkipLine 	bool
+	Targets		string
+}
+
+var prQueueCmd = &cobra.Command{
+	Use:          "queue",
+	Short:        "queue a pr for the current branch",
+	SilenceUsage: true,
+	Args:         cobra.NoArgs,
+	// error or reterr from emperror.dev/errors here?
+	RunE: func(cmd *cobra.Command, args []string) error {
+		variables, err := getMutationVariables()
+		if err != nil {
+			return err
+		}
+
+
+		// I have a feeling this would be better written inside of av/internals
+		client := avgql.NewClient()
+
+		var mutation struct {
+			QueuePullRequest struct {
+				UpdateMergeQueueStatusPayload struct {
+					PullRequest	graphql.String
+				}
+					
+			} `graphql:"githubRepository(input: {owner: $repoOwner, name:$repoName})"`
```

Comment:
you can just pass the `input` as a `map[string]any` fyi, but either way works.

```go
variables := map[string]any{
  "input": map[string]any{
    "owner": ...,
    "name": ...,
  },
}
```

Reply:
wait is this the right annotation here? I think would generate a mutation that looks like

```
mutation (...) {
  githubRepository(input: {...}) {...}
}
```
rather than
```
mutation (...) {
  queuePullRequest(input: {...}) {...}
}
```

Reply:
Fixed